src/FastRoute/src/DBWrapper.cpp

```diff
@@ -1218,13 +1206,13 @@ int DBWrapper::checkAntennaViolations(const std::vector<FastRoute::NET>* routing
     }
     wireEncoder.end();
 
-    odb::orderWires(dbNets[netName], false, false);
+    odb::orderWires(_dbNets[netName], false, false);
 
     std::vector<VINFO> netViol
-        = arc->get_net_antenna_violations(dbNets[netName]);
+        = arc->get_net_antenna_violations(_dbNets[netName]);
```

Comment:
use getNetByIdx(netRoute.idx)
factor the net lookup instead of doing it 3 times in a row